### PR TITLE
gcs: Sort objects in scope source pulldown

### PR DESCRIPTION
**Pull request by mlyle**
Mostly just an edit to get familiar with other parts of the codebase, but the
sheer number of data sources was annoying-- it'll be convenient for it to be
sorted.

### CODE REVIEW

File: ground/gcs/src/plugins/scope/scopegadgetoptionspage.cpp [hunk 1]
```diff
@@ -91,20 +91,36 @@ QWidget* ScopeGadgetOptionsPage::createPage(QWidget *parent)
...
         foreach (UAVDataObject* obj, list) {
+	    QString name = obj->getName();
+
             if (obj->isSingleInstance())
             {
-                options_page->cmbUAVObjects->addItem(obj->getName());
+		toAdd.append(name);
             }
-            else if(obj->getName() != options_page->cmbUAVObjects->itemText(options_page->cmbUAVObjects->count()-1))
+            else if(name != previous)
```
Comment (guilhermito) [inline ground/gcs/src/plugins/scope/scopegadgetoptionspage.cpp]: Not part of this PR I know but why are multi instance object limited to a single instance here?

**Comment by mlyle** [comment]
I don't know much about multi instance objects, but all that's currently stored in the pulldown is the name.  So you'd have several of the object in a row, and the system would be unable to determine which you picked without further work.

**Comment by lilvinz** [comment]
jenkins: test this please

File: ground/gcs/src/plugins/scope/scopegadgetoptionspage.cpp [hunk 2]
```diff
@@ -91,20 +91,36 @@ QWidget* ScopeGadgetOptionsPage::createPage(QWidget *parent)
...
+
+            previous = name;
         }
     }
 
+    qSort(toAdd);
+
+    foreach (QString elem, toAdd) {
+	// Before we only added items to the spectrogram if they were not
+	// single instance; assuming that was in error
```
Comment (kubark42) [inline ground/gcs/src/plugins/scope/scopegadgetoptionspage.cpp]: FYI, we only added multi-instance items because each instance was one bin. So we should probably fix this so that the spectrogram only shows UAVO that it can plot, or teach it how to work with single-instance UAVOs (which would be pretty cool!)
